content/03.results.md

```diff
@@ -246,3 +246,21 @@ If the submitter provided cell types, the submitter annotations are compared to
 The results from both comparisons are displayed in a stacked heatmap available in the report, an example of which is shown in Figure {@fig:figS7}.
+
+## Analysis of bulk RNA-seq
+
+Several projects in the ScPCA Portal quantified many samples' expression both at the bulk and single-cell level.
+Previous research has suggested that, compared to bulk RNA-seq, and single-cell RNA-seq may fail to capture certain cell types [@doi: @10.1038/s41587-020-0465-8] due, for example, to technical aspects of library preparation.
+We therefore sought to ask whether we could identify differences in cell type composition between these two modalities.
+For this analysis, we specifically focused on ScPCA projects with solid tumors, considering only non-multiplexed samples with both sequencing modalities and excluding low-quality single-cell libraries.
+In total, we analyzed 97 samples across five projects (`SCPCP00001`, `SCPCP000002`, `SCPCP000006`, `SCPCP000009`, and `SCPCP000017`), all of which were single-nuclei experiments.
+As described in Materials and Methods, we derived pseudobulk expression matrices from the single-nuclei libraries, and we compared their expression to bulk using a series of linear models (one per ScPCA project) predicting bulk from pseudobulk expression, with a random effect controlling for sample (Figure {@fig:fig6}A, Figure {@fig:figS8}A).
+We observed an expected positive relationship between bulk and pseudobulk expression across all projects.
+
+We next performed an overrepresentation analysis to test if cell type marker genes were enriched either modality.
+We calculated the per-gene median of each project's model residuals and identified outliers in the bulk ($>2.5\sigma$) or pseudobulk direction ($<-2.5\sigma$).
+Using a set of cell type marker genes for each consensus cell type, we calculated the odds ratio for each modality as: the odds a cell type marker gene is present in the given modality's outliers compared to other genes.
+Following permutation testing and P-value correction to control the FDR at $5%$, we indeed found that several cell type marker gene sets were overrepresented in the bulk modality, but never in the pseudobulk modality (Figure {@fig:fig6}B, Figure {@fig:figS8}B).
+This in turn suggests that those cell types may be more prevalently detected in bulk RNA-Seq.
+Most notably, for projects `SCPCP000001` (high-grade gliomas) and `SCPCP000002` (low-grade gliomas), several of these overrepresented cell types were not among the single-nuclei consensus cell types themselves (`SCPCP000001`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "glial cell", "pericyte"; `SCPCP000002`: "blood vessel endothelial cell", "extracellular matrix secreting cell", "microvascular endothelial cell"), many of which are expected to be found in the glioma tumor microenvironment [doi: @10.3389/fimmu.2023.1227126].
```

Comment:
I think I would actually cite this paper which specifically talks about the neural component since the majority of these cell types are either neural or stromal, not immune. We don't have a problem identifying immune cells, but it appears that we are missing identificaiton of neural cell types in the single-cell. 

https://academic.oup.com/noa/article/5/1/vdad009/7053170

This paper also has a section for non-immune cells. 
https://www.frontiersin.org/journals/pharmacology/articles/10.3389/fphar.2024.1355242/full

Reply:
Also, I think we want to first discuss more generally all of the cell types that are over represented in bulk as compared to pseudobulk. I would mention things like neurons, astrocytes, and endothelial cells and state that although we see them in pseudobulk (see consensus annotation figure), they seem to be over represented in bulk, meaning that we might be losing some of these cell types when performing single nuclei and enriching for immune cell types over other normal cell types we may see in bulk. 

I think you can keep the sentence that you have currently about those that are found in bulk but not detected in pseudobulk, but I think that should be secondary after you discuss the main overall finding with all the cell types that are over represented.